css/home.css

```diff
@@ -291,12 +291,62 @@ Media Queries (overriding styles for different screen sizes)
 			url("../assets/images/pages/home/mobile/hero-home-mobile.jpg");
 	}
 
+	/* community partnerships */
+	.community-card .text-container {
+		width: 70%;
+	}
+	.community-card img {
+		object-fit: cover;
+		object-position: 70%;
+	}
+
+	.community-card a {
+		padding: var(--spacing-xs) 0;
+		text-align: center;
+		width: 100%;
+	}
+
+	/* global ecosystem */
+	.global-ecosystem .text-container {
+		width: 80%;
+		padding-right: 175px;
+	}
+
+	.global-ecosystem blockquote {
+		min-width: 340px;
+		top: 130px;
+		right: 125px;
+	}
+
 	/* what we do section */
 	.wwd-card-list {
-		gap: var(--spacing-xs);
+		flex-direction: column;
+		gap: var(--spacing-med);
 	}
+
 	.wwd-card {
-		width: calc(33% - var(--spacing-sm));
+		width: 100%;
+	}
+
+	.wwd-card-text {
+		min-height: auto;
+	}
+
```

Comment:
I played around with different widths and positions, but I feel it best to move wwd 768px adjustments to 1024px

Reply:
Good call